### ISSUE TITLE
Shunt connects with 0 MVAr

### ISSUE THREAD

**Issue by vincentbarbesant** [powsybl/pypowsybl#198]
* **Do you want to request a *feature* or report a *bug*?**
This is a feature request.

* **What is the current behavior?**
If i connect a shunt with `network.connect(shunt_name)`, the shunt is effectively connected, but it absorbs 0 MVAr. I can't specifiy the number of connected sections.

* **What is the expected behavior?**
Add a way (maybe columns in the dataframe) : 
  - to get the maximum number of sections of a shunt
  - to change the number of sections connected for a shunt

* **What is the motivation / use case for changing the behavior?**
I can't properly use shunt with the actual behavior

* **Please tell us about your environment:**
  - PyPowsybl Version : 0.8.0
  - OS Version: Windows 10 Entreprise 64-bit (10.0, Build 18363) (18362.19h1_release.190318-1202)

**Comment by sylvlecl** [comment]
#200 has been included in release 0.9.0.

You should now be able to connect and set the section count at the same time using a dataframe:
```python
df = pd.DataFrame(index=pd.Series(name='id', data=['SHUNT']),
                  columns=['connected', 'section_count'],
                  data=[True, 1])
network.update_shunt_compensators(df)
```
Also, the maximum section count has been added to the dataframe.

Can we consider the issue solved @vincentbarbesant ?

**Comment by vincentbarbesant** [comment]
Tested it, works for me.

Thanks !
